tools/run_tests/distribtest_targets.py

```diff
@@ -97,7 +97,14 @@ def build_jobspec(self):
           ['test/distrib/csharp/run_distrib_test.sh'],
           environ={'EXTERNAL_GIT_ROOT': '../../..'})
     else:
```

Comment:
Should this not be `elif self.platform == 'windows':`, with the `else:` that raises an exception retained?